manifests/repository/ubuntu.pp

```diff
@@ -0,0 +1,18 @@
+# == Class riak::repository::debian
+#
+# Install the correct package repository for Debian
+#
+class riak::repository::ubuntu {
+  ensure_packages('apt-transport-https') # packagecloud is https-only
```

Comment:
edit: I used it before. I feel dumb. I still disagree, even if it's with my past self, but we can merge it.